spec/seedData.test.js

```diff
@@ -0,0 +1,29 @@
+const seedDataTest = require('../data/seedDataGenerator');
+
+describe('Seed Data Generator Function Testing', () => {
+  test('randomElement should be a function', () => {
+    expect(typeof seedDataTest.randoEl).toBe('function');
+  });
+
+  test('getRandomInt should be a function', () => {
+    expect(typeof seedDataTest.randoInt).toBe('function');
+  });
+
+  test('getOneRandomInt should be a function', () => {
+    expect(typeof seedDataTest.oneRandoInt).toBe('function');
+  });
+
+  test('randomDescription should be a function', () => {
+    expect(typeof seedDataTest.randoDes).toBe('function');
+  });
+
+  test('mapping should be a function', () => {
+    expect(typeof seedDataTest.mappingFunc).toBe('function');
+  });
+  test('mapping should return an array', () => {
+    expect(Array.isArray(seedDataTest.mapp)).toBe(true);
```

Comment:
Cool thing you can do here instead of using `Array.isArray()`:

`expect(seedDataTest.mapp).toBeInstanceOf(Array);` 

This can work with Objects and other similar structured data sets. Definitely makes the code look nicer.